### ISSUE TITLE
Fragment inference

### ISSUE THREAD

**Issue by kimbss470** [Pointcept/Pointcept#135]
Hello. I noticed that in ```test mode```, each frame is processed by multiple times which called 'fragment inference'. 
In this case, the points in the voxel which has ```< count.max()``` points are inferenced by multiple times. 
And each results affects the final result (e.g., miou). 
Why didn't you inference only one time and then simply map each voxel's predicted label to points in the voxel?

**Comment by Gofinge** [comment]
Hi, simply mapping each downsampled point to the original resolution might have a negative influence on the final performance. According to the issue you mentioned, we do have a solution mentioned here (https://github.com/Pointcept/Pointcept/issues/108#issuecomment-1893709522) and I have verified that it won't have a negative influence compared with our fragment inference

**Comment by kimbss470** [comment]
Thank you for the comment !

**Comment by RayYoh** [comment]
> Hi, simply mapping each downsampled point to the original resolution might have a negative influence on the final performance. According to the issue you mentioned, we do have a solution mentioned here ([#108 (comment)](https://github.com/Pointcept/Pointcept/issues/108#issuecomment-1893709522)) and I have verified that it won't have a negative influence compared with our fragment inference

Hello, Xiaoyang. Recently I am writing an Instance Tester refers to `SemSegTester` and `InsSegEvaluator`. Does it mean using Inverse mapping downsampled point to the original resolution also has a negative influence on the final performance? In [issue 40](https://github.com/Pointcept/Pointcept/issues/40#issuecomment-1598070945) I noticed that you mentioned InsSegEvaluator is precise which directly uses knn query to get idx.

**Comment by Gofinge** [comment]
Yes, I did not design a better test strategy for instance segmentation. Yet you can trust inverse mapping can at least provide an accurate evaluation of the final performance.

**Comment by xiaosa269** [comment]
@Gofinge Hello, I would like to test on semanticKITTI, and I noticed that there are no labels provided in the test set. It seems that we need to upload the label files generated during the inference process to obtain the final results. However, I found that Fragment inference takes a significant amount of time during testing.

I would like to ask if skipping Fragment inference during testing would affect the generated label file results, and how can I disable it?

Thank you very much for your response!
